substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -286,6 +286,16 @@ public static final long getTearDownFailureNanos() {
         protected void onValueUpdate(EconomicMap<OptionKey<?>, Object> values, String oldValue, String newValue) {
             if ("llvm".equals(newValue) && JavaVersionUtil.JAVA_SPEC > 8) {
                 EmitStringEncodingSubstitutions.update(values, false);
+
+                if ((Boolean) values.get(SpawnIsolates)) {
+                    System.out.println("Isolates are currently not supported by the LLVM backend. The build will continue with isolates disabled.");
```

Comment:
What's the preferred way of emitting this warning?

Reply:
`com.oracle.svm.hosted.SVMHost#warn` If you have access to it. Otherwise, just use `System.out.println("warning: ")`.